section4/exercises/classMethods.js

```diff
@@ -28,13 +28,39 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+
+  changeProtein() {
+    this.protein = "Mushroom";
```

Comment:
Right now, this method will only ever change the protein to a value of "Mushroom". I suggest leveraging an argument in this method like you have in some of the others you've written, so new info can be passed in as the method is called.